app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,27 @@
+class OrdersController < ApplicationController
+  before_action :load_shop, only: :index
+  before_action :check_owner, only: :index
+
+  def index
+    @orders = @shop.all_orders.page(params[:page
+      ]).per(Settings.length.digit_10)
```

Comment:
```suggestion
    @orders = @shop.all_orders
      .page(params[:page]).per(Settings.length.digit_10)
```
xún dòng mới j khó nhìn ghê ri cậu

Reply:
Dạ em có để như chị nhưng k pass đc rubopcop nên mới chuyển thành thế đó ch